### PR TITLE
PC-942: Update dependencies

### PR DESCRIPTION
**Pull request by ulfryk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
     "build": "rm -rf ./lib && node_modules/.bin/tsc --project tsconfig.publish.json && ./scripts/copy-sass-files.sh",
-    "lint": "node_modules/.bin/tslint --project tsconfig.tslint.json --type-check --format verbose './src/**/*.{ts,tsx}'",
+    "lint": "node_modules/.bin/tslint --project tsconfig.tslint.json --format verbose './src/**/*.{ts,tsx}'",
     "prepublish": "yarn build",
     "sandbox": "node_modules/.bin/webpack-dev-server --watch --content-base sandbox/ --inline --config config/webpack.sandbox.js",
-    "test": "yarn check --integrity && yarn outdated && yarn lint && yarn build"
+    "test": "yarn check --integrity && yarn lint && yarn build"
```
Comment (ulfryk) [inline package.json]: Cannot update `react-bootstrap` to latest version…

File: package.json [hunk 2]
```diff
@@ -38,51 +38,51 @@
...
     "jquery": "^3.2.1",
     "monet": "0.9.0-alpha.4",
     "node-sass": "4.5.3",
-    "postcss-loader": "2.0.6",
-    "react": "^15.6.1",
-    "react-bootstrap": "^0.31.3",
-    "react-dom": "^15.6.1",
+    "postcss-loader": "2.0.8",
+    "react": "^16.0.0",
+    "react-bootstrap": "0.31.3",
```
Comment (ulfryk) [inline package.json]: …because it breaks build.

File: yarn.lock [hunk 3]
```diff
@@ -3617,9 +3640,9 @@ ripemd160@^2.0.0, ripemd160@^2.0.1:
     hash-base "^2.0.0"
     inherits "^2.0.1"
 
-rxjs@^5.4.3:
-  version "5.4.3"
-  resolved "https://registry.yarnpkg.com/rxjs/-/rxjs-5.4.3.tgz#0758cddee6033d68e0fd53676f0f3596ce3d483f"
+rxjs@^5.4.3, rxjs@^5.5.1:
```
Comment (ulfryk) [inline yarn.lock]: This one is done by hand, because `yarn` thinks that for `rxjs@^5.4.3` it should load `5.5.0` ;(

Comment (dumpstate) [reply yarn.lock]: I think that how `^` works. Use `~` for patch-level changes.

Comment (ulfryk) [reply yarn.lock]: `^5.4.3` means `>=5.4.3 <6.0.0` so it should resolve to `5.5.1` which is latest, not to `5.5.0`.

https://github.com/npm/node-semver#ranges

Comment (dumpstate) [reply yarn.lock]: I thought you was surprised that `^5.X.Y`, resolved to something with different `X`.

**Pull request merged**